### DISCUSSION TITLE
"Import your data" feature with MariaDb

### DISCUSSION THREAD

**Discussion by Julienh** [planetscale/discussion#197] [Q&A] [answered]
Hello,

I wonder if the "Import your data" feature feature works with a MariaDb database please?

Regards,
Julien

**Comment by jgreet** [comment, marked as answer]
@Julienh it should work but we haven't found it to be as reliable as importing from MySQL. 

If you try it and run in to trouble, we'd appreciate it if you could [open a support ticket](https://support.planetscale.com/hc/en-us/requests/new) and leave the unfinished import running so we can investigate.

**Comment by Julienh** [comment]
@jgreet Thanks, even if the engine storage is different? I'm using Aria, so I guess it will be Innodb during the import (that's what I would like)
